Atomic/atomic.py

```diff
@@ -763,12 +773,21 @@ def get_col_lengths(_images):
             return
 
         _images = self.get_images()
+
+        used_image_ids = [x['ImageID'] for x in self.get_containers()]
+
         if len(_images) >= 0:
             _max_repo, _max_tag = get_col_lengths(_images)
-            col_out = "{0:" + str(_max_repo) + "} {1:" + str(_max_tag) + \
-                      "} {2:12} {3:19} {4:10}"
-            self.write_out(col_out.format("REPOSITORY", "TAG", "IMAGE ID",
-                                         "CREATED", "VIRTUAL SIZE"))
+            col_out = "{0:3} {1:3} {2:" + str(_max_repo) + "} {3:" + str(_max_tag) + \
+                      "} {4:14} {5:18} {6:14}"
+            self.write_out("")
+            self.write_out(col_out.format(Highlight.BOLD + "",
+                                          " ",
```

Comment:
I'm going to remove the bold highlight.  If space is important, this too wastes space in odd ways.  I'll tighten up the indicating.  From here on out, '>' will indicate a used image.

```
   REPOSITORY                                                     TAG      IMAGE ID       CREATED            VIRTUAL SIZE  
*> <none>                                                         <none>   4f8d1487490e   2016-05-29 07:42   502.43 MB     
   tools                                                          latest   2b51dc540df2   2016-05-29 07:36   517.72 MB     
   atomic-test-secret                                             latest   966ce2558544   2016-05-27 19:19   68.0 B        
   fedora_firefox                                                 latest   8904f687ca66   2016-05-24 10:57   925.42 MB     
 > docker.io/blackducksoftware/atomic                             latest   3c9cffdc97fe   2016-05-23 05:16   124.13 MB    
```